djangocms_unitegallery/models.py

```diff
@@ -102,6 +103,9 @@ def __str__(self):
             return self.title
         return _("Photo %s") % self.image.name
 
+    def is_portrait(self):
+        return self.image.height_field > self.image.width_field
+
```

Comment:
Have you tested this ? I really doubt it will work, height_field and width_field are strings, see https://docs.djangoproject.com/en/1.9/ref/models/fields/#imagefield